CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [3.16.1](https://github.com/Backbase/stream-services/compare/3.16.0...3.16.1)
```

Comment:
It's a feature branch it will be bumped to the next 3.x number. By the way, in the meantime, I merged my PR so the next version should be 3.18 I think.

Reply:
Ok thanks, I will change it to 3.18.